### ISSUE TITLE
Uncaught Error: AG Grid - Illegal version string: 27.1.0 (RestoreChart)

### ISSUE THREAD

**Issue by HuseIstvan** [ag-grid/ag-grid-enterprise#280]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-enterprise/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

When restoring a chart model using gridApi.restoreChart(), which was saved by the gridApi.getChartModels(), we run into an exception. Looks like the version string is incorrect perhaps?

```
Uncaught Error: AG Grid - Illegal version string: 27.1.0
    at versionParts (ag-grid-enterprise.cjs.js:50393:1)
    at upgradeChartModel (ag-grid-enterprise.cjs.js:50384:1)
    at ChartService.restoreChart (ag-grid-enterprise.cjs.js:50486:1)
    at GridApi.restoreChart (ag-grid-community.cjs.js:30841:1)
    at onClick (DetailsTable.js:653:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:4161:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4210:1)
    at invokeGuardedCallback (react-dom.development.js:4274:1)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:4288:1)
    at executeDispatch (react-dom.development.js:9038:1)
```

When I click on the error to get more information, this is the piece of code I see in my Chrome Devtools:
```

// the line below is automatically modified during releases - do not modify
// (see scripts/release/updateChartModel.js)
var CURRENT_VERSION = "27.2.1";
function upgradeChartModel(model) {
    if (model.version == null) {
        // First release with version field.
        model.version = "27.1.0";
    }

    var _a = versionParts(model.version);
    // TODO: Add some transforms as the model changes over time.
    // if (major < 28) {
    //     model = migrateToV28Model(model);
    // }
    return model;
}
function versionParts(version) {
    if (!/^([:digits:]{1,2}\.){2}[:digits:]{1}$/.test(version)) {
        throw new Error('AG Grid - Illegal version string: ' + version);
    }
    var split = version.split('.')
        .map(function (v) { return Number(v); });
    return {
        major: split[0],
        minor: split[1],
        patch: split[2],
    };
}
```

![2022-05-12 11_28_13-Window](https://user-images.githubusercontent.com/33901174/168039606-2b776851-2609-4e46-a5f0-8596b84e2a14.jpg)
![2022-05-12 11_28_31-Window](https://user-images.githubusercontent.com/33901174/168039610-603dcb50-8d64-4e21-be30-d87c7f6bb2fc.jpg)
![2022-05-12 11_29_12-Window](https://user-images.githubusercontent.com/33901174/168039613-4f628f30-f00d-4027-acfe-0b20af65f2f2.jpg)

**Expected behavior**

The chart should show. (Based on the chartModel that is being given to it)

**Minimal reproduction of the problem with instructions**

- I create a range chart from the AG Grid table. (User Created chart)
- I have a button, which saves the chartModel with gridApi.getChartModels()
- I also see this object in my console, which I can copy, and paste it into another button, where I can try to load it in.
- I click the (restore)button, which holds my chartModel object, but it runs into an exception (see exception above).

**What is the motivation / use case for changing the behavior?**

Saving and loading in user created charts.

**Please tell us about your environment:**

- Windows 10
- Visual Studio Code
- NPM
- localhost
- React 18

* **ag-Grid-Enterprise version:** 2.0.X

"ag-grid-community": "^27.2.1",
"ag-grid-enterprise": "^27.2.1",
"ag-grid-react": "^27.2.1",
"react": "^18.1.0",
"react-dom": "^18.1.0",

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chrome Version 101.0.4951.64 (Official Build) (64-bit)

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
all (ES6)

**Comment by kiril-matev** [comment]
Hello @HuseIstvan!

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:
**AG-6772 [Charts] When restoring integrated charts, an exception is thrown because of a version parsing issue**

This issue will be resolved in our next version to be released before May 20th. Upgrade to this version once it's available to resolve the issue.

**Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-XXXX) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks again for bringing this up with us - we appreciate your time!
